packages/docs/src/pages/tutorial/component/inline/index.mdx

```diff
@@ -3,8 +3,19 @@ title: Light Component
 layout: tutorial
 ---
 
-Qwik components are lazy-loadable. This is achieved through `component$()` method. The presence of `$` crates a lazy-loaded boundary. The `$` will be discussed later. Qwik also supports inline components. Inline components are not by themselves lazy loadable and instead load as part of the parent component where they are used. (Inline components are equivalent to regular components in most other frameworks.)
+One of Qwik's super powers lies in its lazy-loading features. 
 
-In this example we have pre-declared an `<App>` and a `<Greeter>` component. The `<Greeter/>` component is currently a Qwik Component because it is declared with `component$()`. Remove the `component$()` to convert `<Greeter>` to inline component. Inline components do not have host elements, (described later.)
+Qwik lazy-loads:
 
-Open the `Symbols` tab and notice that the `<Greeter/>` component is no longer an independent export, and instead it is bundled as part of the `<App>` component.
+- **Independent of DOM presence**: Components can load before and after they are added to the DOM.
```

Comment:
```suggestion
```
Not sure what you were going for, but the above does not make sense without the context of Qwik. So I would just remove it.

Reply:
It was an attempt to work with this comment:

> Lazy-loadable=> But all frameworks claim that => Qwik can lazy-load even the components that are in the current DOM (other frameworks can only lazy-load components that are currently not in the DOM)

perhaps there's another way to word it?